Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1306,6 +1441,39 @@ def get_conversation_by_name(conversation_name: str) -> dict:
         else:
             conversations = [conversation]
         set_to_integration_context_with_retries({'conversations': conversations}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
+
+
+def get_conversation_by_name(conversation_name: str) -> dict:
+    """
+    Get a slack conversation by its name. Order of operation is:
+    1. Check the COMMON_CHANNEL parameter for the conversation
+    2. Check the integration context for the conversation
+    3. If SAFE_MODE is disabled, then we will paginate the api
+    4. If SAFE_MODE is disabled, then we will save the results of the pagination to context
+
+    Args:
+        conversation_name: The conversation name
+
+    Returns:
+        The slack conversation
+    """
+
+    conversation_to_search = conversation_name.lower()
+    conversation: dict = {}
+    # Checks if the channel is defined in the integration params
+    if len(COMMON_CHANNELS) > 0:
+        conversation = search_conversation_in_params(conversation_to_search)
+
+    # Find conversation in the cache
+    if not conversation:
+        conversation = search_conversation_in_context(conversation_to_search)
```

Comment:
if we are on safe mode, we should not save conversations to context and there for should not search for it in context.
please search if we are saving it to context if we are not on safe mode.
this shuld be the same with users.

Reply:
still relevant